content/blog/2025/how-python-has-shaped-our-community.md

```diff
@@ -0,0 +1,31 @@
+Title: How Python has shaped our community. How PyCon India is a big promotor of same
```

Comment:
The title is ambiguous. What is PyCon India a big promoter of? Python? Or, the community?